docs/tutorials/uniq-factories/creating-uniq-factories/how-to-create-uniq-factory-using-toolkit.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'How to create a Uniq Factory using the Ultra Toolkit'
+order: 4
+outline: [0, 4]
+---
+
+# How to create a Uniq Factory using the Ultra Toolkit
+
+![](/images/token-factories/new-token-factory.png)
+
+## Prerequisites
+
+-   You must have created and uploaded the uniq metadata files. See [How to create Uniq Metadata](./how-to-create-uniq-metadata.md) for more information.
+-   Your account must have sufficient UOS tokens for transactions and fees. Refer to [Tutorial - Using the Faucet and Buying RAM on Ultra Testnet](../../fundamentals/tutorial-obtain-token-and-purchase-ram.md#obtaining-uos-tokens-using-the-faucet) for more information.
+
+## Goal
+
+The goal of this guide is to help you create your first Uniq Factory on the Ultra testnet network.
```

Comment:
Better to capitalize Testnet for consistency, it is usually starts with a capital